extensions/xcode/src/components/xcode-simulator-list/xcode-simulator-list-dropdown.component.tsx

```diff
@@ -0,0 +1,21 @@
+import { List } from "@raycast/api";
+import { XcodeSimulatorStateFilter } from "../../models/xcode-simulator/xcode-simulator-state-filter.model";
+
+export function XcodeSimulatorStateDropdown(props: {
+  onSimulatorStateFitlterChange: (newValue: XcodeSimulatorStateFilter) => void;
+}) {
+  const { onSimulatorStateFitlterChange } = props;
+  return (
+    <List.Dropdown
+      tooltip="Select a simulator state"
+      storeValue={true}
+      onChange={(newValue) => {
+        onSimulatorStateFitlterChange(newValue as XcodeSimulatorStateFilter);
```

Comment:
Typo `onSimulatorStateFitlterChange` should be changed to `onSimulatorStateFilterChange`

Reply:
✅ Fixed